compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -698,6 +698,30 @@
                 }
             ]
         },
+        {
+            "name": "ErrorConstructorExpressionNode",
```

Comment:
Shall we rename this as FunctionalConstructorExpression?

Reply:
Done